www/index.html

```diff
@@ -55,6 +55,33 @@ <h1>PhoneGap</h1>
     <script type="text/javascript">
         app.initialize();
     </script>
-</body>
+</body> -->
+
+<body>
+    My user ID: <input id="myUserId"><br>
+    <hr>
+    Enter user ID you want to call<input id="targetUserId">
+    <button id="callUser">Call!</button>
+    <div id="debug">
+        Debug Messages:
+    </div>
+
+    <span>
+      <h3>Remote video</h3>
+      <video id="remotevideo" autoplay style="border: 5px solid black"/>
+    </span>
+    <span>
+      <h3>Local video</h3>
+      <video id="localvideo" autoplay style="border: 5px solid black"/>
+    </span>
+  
+    <script type="text/javascript" charset="utf-8" src="cordova.js"></script> 
+    <script src="https://www.gstatic.com/firebasejs/4.5.0/firebase.js"></script>
+    <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
Use
cordova.plugins.iosrtc.registerGlobals
Now you have window.RTCPeerConnection, navigator.getUserMedia, etc.



See:
https://www.npmjs.com/package/cordova-plugin-iosrtc

// Just for Cordova apps. 
document.addEventListener('deviceready', function () {
  // Just for iOS devices. 
  if (window.device.platform === 'iOS') {
    cordova.plugins.iosrtc.registerGlobals();